data/common.yaml

```diff
@@ -1,50 +1,50 @@
 ---
-elasticsearch::ensure: present
-elasticsearch::api_basic_auth_password: ~
-elasticsearch::api_basic_auth_username: ~
-elasticsearch::api_ca_file: ~
-elasticsearch::api_ca_path: ~
+elasticsearch::api_basic_auth_password:
```

Comment:
if something is undef, it doesn't need to be added to hiera at all

Reply:
> if something is undef, it doesn't need to be added to hiera at all

Not without adding `= undef,` to the Class-Parameter. 
Anyhow.  This is not Subject, of this PR.